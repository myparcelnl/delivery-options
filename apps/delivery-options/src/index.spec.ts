import * as index from './index';
import {describe, expect, it} from 'vitest';

describe('exports', () => {
  it('exports from index.ts', () => {
    expect(Object.keys(index).sort()).toMatchSnapshot();
  });
});
