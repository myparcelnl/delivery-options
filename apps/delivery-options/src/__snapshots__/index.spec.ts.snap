// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`exports > exports from index.ts 1`] = `
[
  "AddressField",
  "CaretLeftIcon",
  "CaretRightIcon",
  "CarrierBoxLoader",
  "CarrierLogoLoader",
  "CarrierSetting",
  "CheckboxGroupInput",
  "ConfigSetting",
  "CustomDeliveryType",
  "DATES_SHOWN_MD",
  "DATES_SHOWN_SM",
  "DELIVERY_MOMENT_PACKAGE_TYPES",
  "DISABLE_DELIVERY_OPTIONS",
  "DoButton",
  "EcoFriendlyIcon",
  "EcoFriendlyLabel",
  "FIELD_DELIVERY_DATE",
  "FIELD_DELIVERY_MOMENT",
  "FIELD_HOME_OR_PICKUP",
  "FIELD_PICKUP_LOCATION",
  "FIELD_SHIPMENT_OPTIONS",
  "GroupInput",
  "GroupInputLoader",
  "HIDE_DELIVERY_OPTIONS",
  "HOME_OR_PICKUP_HOME",
  "HOME_OR_PICKUP_PICKUP",
  "IconButton",
  "Icons",
  "InfoLabel",
  "KEY_ADDRESS",
  "KEY_CARRIER_SETTINGS",
  "KEY_CONFIG",
  "KEY_INITIAL",
  "KEY_STRINGS",
  "LeafletMap",
  "LeafletMapInner",
  "LeafletMarker",
  "MAP_MARKER_CLASS_ACTIVE",
  "MAP_MARKER_CLASS_PREFIX",
  "MyParcelDeliveryOptions",
  "OUTPUT_EVENT_DEBOUNCE_DELAY",
  "ParcelLockerIcon",
  "PickupLocationsView",
  "PriceTag",
  "RENDER_DELIVERY_OPTIONS",
  "RadioButtonLoader",
  "RadioGroupInput",
  "SHOWN_OPENING_HOURS",
  "SHOWN_SHIPMENT_OPTIONS",
  "SHOW_DELIVERY_OPTIONS",
  "SUPPORTED_DELIVERY_TYPES",
  "SUPPORTED_PACKAGE_TYPES",
  "SUPPORTED_PLATFORMS",
  "SUPPORTED_SHIPMENT_OPTIONS",
  "UNSELECT_DELIVERY_OPTIONS",
  "UPDATED_ADDRESS",
  "UPDATED_DELIVERY_OPTIONS",
  "UPDATE_CONFIG_IN",
  "UPDATE_DELIVERY_OPTIONS",
  "bootDeliveryOptions",
  "calculateCutoffTime",
  "calculateDropOffDays",
  "calculatePackageType",
  "createCarrierMarkerIcon",
  "createDeliveryTypeTranslatable",
  "createFakeShipmentOptions",
  "createGetDeliveryOptionsParameters",
  "createLatLngParameters",
  "createNextDate",
  "createPackageTypeTranslatable",
  "createRelativeDateFormatter",
  "createUtcDate",
  "getConfigFromWindow",
  "getConfigPriceKey",
  "getDefaultAddress",
  "getDeliveryTypePrice",
  "getResolvedCarrier",
  "getResolvedDeliveryType",
  "getResolvedValue",
  "hasSlotContent",
  "padTime",
  "parseJson",
  "showDeveloperInfo",
  "stringToDate",
  "toTime",
  "useActiveCarriers",
  "useAddressStore",
  "useBreakpoints",
  "useConfigStore",
  "useCurrentPlatform",
  "useDateFormat",
  "useDeliveryOptionsForm",
  "useDeliveryOptionsIncomingEvents",
  "useDeliveryOptionsOutgoingEvents",
  "useFeatures",
  "useFormatDistance",
  "useLanguage",
  "useMostEcoFriendly",
  "useOptionsGroupedByCarrier",
  "usePickupLocation",
  "usePickupLocationsMap",
  "useProvideElementWidth",
  "useResolvedCarrier",
  "useResolvedDeliveryDates",
  "useResolvedDeliveryMoments",
  "useResolvedDeliveryOptions",
  "useResolvedPickupLocations",
  "useResolvedValues",
  "useSelectedDeliveryMoment",
  "useSelectedPickupLocation",
  "useSelectedValues",
  "useShipmentOptionsOptions",
  "useTimeRange",
]
`;
